src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    @Override
+    public List<FruitTransaction> convertListToObjects(List<String> list) {
+        List<FruitTransaction> listOfTransactions = new ArrayList<>();
+        for (int i = 1; i < list.size(); i++) {
+            String type = list.get(i).substring(0, list.get(i).indexOf(","));
```

Comment:
Using `String#substring` and `String#indexOf` directly within the loop for parsing can make the code harder to read and maintain. Consider creating a separate method or using a library to parse CSV lines into objects.